au3/libraries/au3-cloud-audiocom/sync/CloudProjectsDatabase.cpp

```
@@ -919,6 +930,12 @@ bool CloudProjectsDatabase::OpenConnection()
     return RunMigrations();
 }
 
+void CloudProjectsDatabase::CloseConnection()
```

Cppcheck (2.20.0)

[style] The function 'CloseConnection' is never used.

(unusedFunction)